tests/test_default.py

```diff
@@ -0,0 +1,21 @@
+def test_brew_taps(Command):
+    c = Command('/usr/local/bin/brew tap --list')
+    assert c.rc == 0
+    tapped = c.stdout.split()
+    assert 'homebrew/core' in tapped
```

Comment:
Forgive me if I'm being dense... but wouldn't this just be testing that Ansible's core modules are doing what they say they do? The role just tells Ansible to verify the taps are tapped and the packages are installed—if that was broken, then the role's normal tests (run it, run it again) would fail.

I guess I'm failing to see why I'd want to add a second layer of tests to this role which simply asserts that the tests Ansible itself runs during the playbook run are passing? Like checking if `yes == yes` actually equals `yes == yes`... if it doesn't, that's an upstream issue and I'd start looking at an alternative solution besides Ansible if I can't trust it to at least verify something is installed or not :/